### PR TITLE
Add Polish retail bonds source

### PR DESCRIPTION
**Pull request by mkopec87**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
         "jupyter",
         "jupytext",
         "seaborn",
+        "xlrd",
```
Comment (adri0) [inline setup.py:34]: Why do we need to import this library? `openpyxl` doesn't work for loading Polish bonds xls?

File: investd/sources/bonds.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        df = df[df[COL_STATUS] == STATUS_COMPLETED]
+
+        # correct the number and amount of premature sell operations
+        for i in df.index[df[COL_TYPE] == TYPE_PREMATURE_SELL].tolist():
+            df.at[i, COL_COUNT] = df.at[i, COL_AMOUNT] // BOND_VALUE
+            df.at[i, COL_AMOUNT] = (
+                df.at[i, COL_AMOUNT]
+                + df.at[i + 1, COL_AMOUNT]
+                + df.at[i + 2, COL_AMOUNT]
+            )
```
Comment (adri0) [inline investd/sources/bonds.py:41]: This is interesting. I wonder why it' listed like that in the file. Pretty cool `for statement` though.

**Review by adri0 — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (mkopec87) [reply setup.py:34]: It only supports XLSX, not XLS :(

File: investd/sources/bonds.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+    def _convert(self, record: pd.Series) -> Transaction:
+        quantity = record[COL_COUNT]
+        amount = record[COL_AMOUNT]
+        price = amount / quantity
+
+        if record[COL_TYPE] == TYPE_BUY:
+            action = "BUY"
+        elif record[COL_TYPE] in {TYPE_SELL, TYPE_PREMATURE_SELL}:
+            action = "SELL"
```
Comment (mkopec87) [inline investd/sources/bonds.py:72]: should the amount be negative when we sell?

Comment (adri0) [reply investd/sources/bonds.py:72]: Not necessary. Currently, the report I created convert SELLs to negative. We can change later if it's more convenient having negative amounts in the source parsers.

**Review by adri0 — APPROVED** [review]

**Pull request merged**